lib/explorer/backend/lazy_series.ex

```diff
@@ -60,6 +60,7 @@ defmodule Explorer.Backend.LazySeries do
     distinct: 1,
     unordered_distinct: 1,
     slice: 3,
+    slice_by_indices: 2,
```

Comment:
Shouldn’t this still be `slice: 2`? I thought that we try to keep the same names as the callbacks as much as possible.

Reply:
@josevalim The problem is that I cannot have a function with different arities because we are defining functions in the NIF module with this.
But I think I can control that elsewhere. Let me try..

Reply:
I think this solves the problem: https://github.com/elixir-nx/explorer/pull/531/commits/2b14c2946f5408991d4e15055aa2d5e108a6aebd